src/@crema/components/AppPage/index.tsx

```
@@ -0,0 +1,261 @@
+        loading: !!(listQuery?.isFetching || listQuery?.isLoading),
+        data: listQuery?.data, // PaginatedAPIResponse<any>
+      };
+    }, [listQuery?.isFetching, listQuery?.isLoading, listQuery?.data]);
+
+    // ---------- Adapter mutation cho AppAddEditModal ----------
+    // AppAddEditModal gọi operation({ payload, onSuccess, onError })
+    const createAdapter = endpoint
+      ? (() => {
+          const m = baseApi.useCreateItem();
+          return async ({
+            payload,
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)